robotpy_build/command/build_pyi.py

```diff
@@ -159,6 +140,68 @@ def main():
         cfg["out"],
     ] + cfg["stubs"]
 
+    math_to_add = {}
+
+    class fp_wrapper:
+        def __init__(self, fp) -> None:
+            self.fp = fp
+        
+        def write(self, s: str) -> None:
+            # fix underscored names
+            s = re.sub(r"([a-zA-Z0-9_]+)\._\1", r"\1", s)
```

Comment:
What is this for?

Reply:
When wpi* depends on wpiutil for example, `import wpiutil._wpiutil` is added to the top of the stubs. This changes that to `import wpiutil`. I haven't tested this yet but I'm hoping that it tricks intellisense into providing better imports.

Right now, when my students write code and let vscode auto import stuff, it always goes all the way to the underscored packages.

Reply:
This feels like a really blunt instrument for doing that.